helpers/pluck.js

```diff
@@ -1,10 +1,25 @@
 'use strict';
-
-var _ = require('lodash');
+const common = require('./lib/common.js');
 
 const factory = () => {
     return function(collection, path) {
-        return _.pluck(collection, path);
+        const arr = [];
```

Comment:
What is the current behavior of this function? Should it return an object when passed an object, or is it supposed to always return an array?

Reply:
.pluck is now map in lodash and returns a mapped array.  I used both _.pluck and my own against test cases and they pass.

Reply:
I think it would be better to keep lodash for collection helpers.